src/clj_cbor/core.clj

```diff
@@ -78,15 +78,6 @@
 
 ;; ## Encoding Functions
 
-(defn- data-output-stream
```

Comment:
I think it'd be better to keep this function and do something similar to https://github.com/greglook/clj-cbor/commit/760124f5ed31b8d554e3a414422179db2e2a9741 - if the argument is already a `DataOutputStream` then return it, if it is an `OutputStream` already then wrap it directly, otherwise use `io/output-stream` to coerce it before wrapping.